packages/node_modules/@ciscospark/internal-plugin-presence/src/index.js

```diff
@@ -0,0 +1,85 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+import '@ciscospark/internal-plugin-wdm';
+
+import {registerInternalPlugin} from '@ciscospark/spark-core';
+import {has} from 'lodash';
+
+import Presence from './presence';
+import config from './config';
+
+registerInternalPlugin(`presence`, Presence, {
+  payloadTransformer: {
+    predicates: [
+      {
+        name: `normalizeSingleStatusResponse`,
+        direction: `inbound`,
+        test(ctx, response) {
+          return Promise.resolve(has(response, `body.eventType`));
+        },
+        extract(response) {
+          return Promise.resolve(response);
+        }
+      },
+      {
+        name: `normalizeStatusResponses`,
+        direction: `inbound`,
+        test(ctx, response) {
+          return Promise.resolve(has(response, `body.statusList`) || has(response, `body.responses`));
```

Comment:
Again, are these test specific enough? Should something in the first item of each list be checked to confirm it's the type you expect it to be?